src/commands/output.js

```diff
@@ -71,8 +71,9 @@ class OutputCommand extends TerraformCommand {
 
         results.forEach(it => {
           const stdout = (Buffer.from(it.buffer)).toString('utf8');
-          const json = stdout[0] !== '{' ? stdout.indexOf('{') : stdout;
-
+          const indexStart = stdout.indexOf('{');
+          const json = stdout[0] !== '{' ? stdout.substr(indexStart, stdout.length - indexStart) : stdout;
```

Comment:
`substr` is [deprecated](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/substr), use `substring`

```suggestion
          const json = stdout[0] !== '{' ? stdout.substring(indexStart, stdout.length - indexStart) : stdout;
```